extensions/blocks/contact-form/components/jetpack-newsletter-integration-settings.js

```diff
@@ -44,6 +63,191 @@ const NoConsentBlockSettings = () => {
 	);
 };
 
+const pluginStateEnum = Object.freeze( {
+	ACTIVE: 1,
+	INSTALLED: 2,
+	NOT_INSTALLED: 3,
+} );
+
+const CreativeMailPluginErrorState = ( { error } ) => {
+	return (
+		<Notice isDismissible={ false } status="error">
+			{ jetpackCreateInterpolateElement(
+				__(
+					'The plugin failed to install. <b /> Please check the <a>plugin information</a> for detailed requirements.',
+					'jetpack'
+				),
+				{
+					a: (
+						<ExternalLink href="https://wordpress.org/plugins/creative-mail-by-constant-contact" />
+					),
+					b: <span>{ error }</span>,
+				}
+			) }
+		</Notice>
+	);
+};
+
+const CreativeMailPluginIsInstalling = ( { isActivating } ) => {
+	return (
+		<Button
+			isSecondary
+			icon={ <Icon style={ { animation: 'rotation 2s infinite linear' } } icon="update" /> }
+			disabled
+		>
+			{ ! isActivating && __( 'Installing…', 'jetpack' ) }
+			{ isActivating && __( 'Activating…', 'jetpack' ) }
+		</Button>
+	);
+};
+
+const CreativeMailPluginIsNotInstalled = ( {
+	installAndActivateCreativeMailPlugin,
+	isInstalling,
+} ) => {
+	return (
+		<p>
+			<em>
+				{ __(
+					'To start sending email campaigns, install the Creative Mail plugin for WordPress.',
+					'jetpack'
+				) }
+				<br />
+				<br />
+				{ isInstalling && <CreativeMailPluginIsInstalling /> }
+				{ ! isInstalling && (
+					<Button isSecondary onClick={ installAndActivateCreativeMailPlugin }>
+						{ __( 'Install Creative Mail plugin', 'jetpack' ) }
+					</Button>
+				) }
+			</em>
+		</p>
+	);
+};
+
+const CreativeMailPluginIsInstalled = ( { activateCreativeMailPlugin, isInstalling } ) => {
+	return (
+		<p>
+			<em>
+				{ __(
+					'To start sending email campaigns, active the Creative Mail plugin for WordPress.',
+					'jetpack'
+				) }
+			</em>
+			<br />
+			<br />
+			{ isInstalling && <CreativeMailPluginIsInstalling isActivating /> }
+			{ ! isInstalling && (
+				<Button isPrimary onClick={ activateCreativeMailPlugin }>
+					{ __( 'Activate Creative Mail Plugin', 'jetpack' ) }
+				</Button>
+			) }
+		</p>
+	);
+};
+
+const CreativeMailPluginIsActive = () => {
+	return (
+		<p>
+			<em>
+				{ __( 'You’re all setup for email marketing with Creative Mail.', 'jetpack' ) }
+				<br />
+				<br />
+				<ExternalLink href="/wp-admin/admin.php?page=creativemail">
+					{ __( 'Open Creative Mail', 'jetpack' ) }
```

Comment:
Should this say "Settings"?

```suggestion
					{ __( 'Open Creative Mail settings', 'jetpack' ) }
```